model/infinispan/src/main/java/org/keycloak/connections/infinispan/InfinispanUtil.java

```diff
@@ -56,6 +56,10 @@ public static TopologyInfo getTopologyInfo(KeycloakSession session) {
 
     private static final Object CHANNEL_INIT_SYNCHRONIZER = new Object();
 
+    /**
+     * @deprecated to be removed without replacement.
+     */
+    @Deprecated(since = "26.3", forRemoval = true)
```

Comment:
The deprecated methods in this class (and others) are not used internally anymore. @ahus1 @pruivo  What's the policy for deprecating/removing classes in the model modules. Should all code in this module be considered public API or just interfaces? I don't see a `package-info.java` so it's not clear to me what the expectations are here.

Reply:
It is not clear what is public API and what users are using (I'm aware some users are creating caches for their providers). I'm playing safe and keeping them around and removing them in 26.4 or 26.5.

Reply:
That's fine by me. I see we're now making better use of `impl.*` packages, so in the future hopefully what is/isn't supported will be clearer.